opencti-platform/opencti-graphql/src/utils/filtering.js

```diff
@@ -127,6 +129,74 @@ export const convertFiltersToQueryOptions = async (context, filters, opts = {})
   return { types, orderMode, orderBy: [field, 'internal_id'], filters: queryFilters };
 };
 
+const testRelationFromFilter = (stix, extractedIds, operator) => {
```

Comment:
Can you explain the changes of this file ? Not sure to understand

Reply:
I put some parts of the code in a function because this code is used several time (exemple: testRelationFromFilter is used in case key=instanceId, and in case key=fromId)